src/fsearch_window.c

```diff
@@ -1134,6 +1134,10 @@ fsearch_application_window_update_query_flags(FsearchApplicationWindow *win) {
 
 static FsearchDatabaseIndexType
 get_sort_type_for_name(const char *name) {
+    if (!name) {
+        g_warning("[get_sort_type_for_name] name is nullptr");
+        return 0;
```

Comment:
Even though the effect is exactly the same, I think it's a better idea to return `DATABASE_INDEX_TYPE_NAME` instead of `0`. It's a little more descriptive, because it's considered the default sorting type.